src/scripts/clipperUI/components/modeButtonSelector.tsx

```diff
@@ -1,28 +1,39 @@
-import {ClientType} from "../../clientType";
 import {Constants} from "../../constants";
-import {Experiments} from "../../experiments";
-
 import {AugmentationHelper} from "../../contentCapture/augmentationHelper";
-
 import {ExtensionUtils} from "../../extensions/extensionUtils";
 import {InvokeMode} from "../../extensions/invokeOptions";
-
 import {Localization} from "../../localization/localization";
-
 import {ClipMode} from "../clipMode";
 import {ClipperStateProp} from "../clipperState";
 import {ComponentBase} from "../componentBase";
-
 import {ModeButton} from "./modeButton";
 
+export interface NoAriaPropsForModeButton {
```

Comment:
ModeButtonSelector should not have ModeButtonProps and similar interfaces. These should be declared in modeButton.tsx

Reply:
Also I wonder if there's a better name for this interface ... it could be clearer